### DISCUSSION TITLE
Jalali calendar is missing

### DISCUSSION THREAD

**Discussion by mahdihadinarvan** [mantinedev/mantine#4470] [Feature requests]
Hi,
I use **jalali calendar** but with **dayjs** set to `'fa'` It only changes month names but calendar stills gregorian for example in jalali calendar we are in year 1402 but when I set `DatesProvider` `locale` to `fa` it still shows 2023

**Comment by rtivital** [comment]
Jalali calendar is not supported. If you need this feature, you can use other dates library that supports it.
